### PR TITLE
Remove upstream issue and PR templates

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -1,38 +0,0 @@
-<!--
```
Comment (wbl) [inline .github/ISSUE_TEMPLATE:1]: These are good questions!

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -1,25 +0,0 @@
-This PR will be imported into Gerrit with the title and first
-comment (this text) used to generate the subject and body of
-the Gerrit change.
-
-**Please ensure you adhere to every item in this list.**
-
-More info can be found at https://github.com/golang/go/wiki/CommitMessage
-
-+ The PR title is formatted as follows: `net/http: frob the quux before blarfing`
```
Comment (wbl) [inline .github/PULL_REQUEST_TEMPLATE:9]: Keep some of this

**Review by wbl — APPROVED** [review]

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -1,38 +0,0 @@
...
-A link on play.golang.org is best.
--->
-
-
-
-### What did you expect to see?
-
-
-
-### What did you see instead?
```
Comment (claucece) [inline .github/ISSUE_TEMPLATE:38]: These are indeed good questions. Perhaps we move then to the wiki or something?

Comment (cjpatton) [reply .github/ISSUE_TEMPLATE:38]: Opened #87 to track this discussion. Are you alright with merging this PR as-is and adding new templates later?

Comment (claucece) [reply .github/ISSUE_TEMPLATE:38]: yes! we can do that ;) Thank you!

**Review by claucece — APPROVED** [review]

**Pull request merged**